tests/TestHelpers/GraphHelper.php

```diff
@@ -1983,4 +1983,63 @@ public static function enableShareSync(
 			\json_encode($body)
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $spaceId
+	 * @param array $shareeIds
+	 * @param array $shareTypes
+	 * @param string|null $permissionsRole
+	 * @param string|null $permissionsAction
+	 * @param string|null $expireDate
+	 *
+	 * @return ResponseInterface
+	 */
+	public static function sendSharingInvitationForDriveRoot(
```

Comment:
`sendSharingInvitationForDrive` should be enough for separation
```suggestion
	public static function sendSharingInvitationForDrive(
```